### DISCUSSION TITLE
disable zero configuration?

### DISCUSSION THREAD

**Discussion by schmitch** [PSAppDeployToolkit/PSAppDeployToolkit#712] [Q&A] [answered]
hello, is there any way to disable the zero configuration of msi files? currently I need to add extra parameters to execute-msi, so thus I want to disable it.

**Comment by That-Annoying-Guy** [comment]
I avoid the issue all together by commenting the  zero config lines in the Deploy-Application.ps1 file.

**Comment by mjr4077au** [comment, marked as answer]
Zero-Config mode is defeated simply by defining $AppName.
